src/item_factory.cpp

```diff
@@ -4095,367 +4004,140 @@ static void replace_materials( const JsonObject &jo, itype &def )
     }
 }
 
-void Item_factory::load_basic_info( const JsonObject &jo, itype &def, const std::string &src )
+void itype::load( const JsonObject &jo, const std::string_view )
 {
-    bool strict = src == "dda";
-
-    restore_on_out_of_scope restore_check_plural( check_plural );
+    std::string src = "dda";
```

Comment:
This is the second to the function. I know other things define it like this, but that's wrong. Use the argument to the function.